tests/test_to_container.py

```diff
@@ -462,3 +472,98 @@ def test_enum_to_str_for_list(
         cfg = OmegaConf.create(src)
         container: List[Any] = OmegaConf.to_container(cfg, enum_to_str=enum_to_str)  # type: ignore
         assert container == [expected]
+
+
+class TestThrowOnMissing:
+    """Tests the `throw_on_missing` arugment to OmegaConf.to_container"""
+
+    @mark.parametrize(
+        "op",
+        [
+            param(
+                lambda cfg: OmegaConf.to_container(cfg, throw_on_missing=True),
+                id="to_container",
+            ),
+            param(OmegaConf.to_object, id="to_object"),
+        ],
+    )
+    @mark.parametrize(
+        "cfg",
+        [
+            # to_container: throw_on_missing
+            param(DictConfig("???"), id="dict,missing"),
```

Comment:
Updated in https://github.com/omry/omegaconf/pull/730/commits/28bcd62c77b1ce1ce44f96d52bd318d0a727a41f. Thanks!

Reply:
I don't really want to make a big deal of this, so I'm resolving this comment, but FYI even when there are more than two items, I think usually `:` is used instead of commas (as you did in 834a0bcff56f84c83e57bb6ec2d6a6bb04b7bfda). One example is:
https://github.com/omry/omegaconf/blob/a2e25ddc7aa34bf904ce6f111accc08f50557e58/tests/test_merge.py#L260

Reply:
Interesting... I hadn't noticed the use of multiple colons before.

I was building from what I'd seen in `test_errors.py`, where there are lots of mixed colon+comma id's (e.g. `id="structured:merge,invalid_field_type"`).
It feels almost like colons are a sort of "nesting" of test-case categories, whereas commas feel like taking an intersection of categories.

Reply:
Makes sense :)